### ISSUE TITLE
Argentina ARS/BTC Market Maker Bounty

### ISSUE THREAD

**Issue by flix1** [bisq-network/growth#172]
### Argentina ARS/BTC Market Maker Bounty

To claim this bounty the contributor must make a market in Bisq ARS/BTC for 1 month.
https://bisq.network/markets/?currency=btc_ars

This will involve:

1. Keeping at least 2 BUY and 2 SELL offers live during 1 month.

2. Attracting other traders to the market through marketing, communication, social media...

3. Completing at least 5 trades.

4. Report with feedback here on the details of the trades (payment method used, problems encountered, suggestions for improvement, fees paid, time it took to complete the trade).

5. Attend at least 1 live Bisq call to answer questions and provide feedback on the experience.

### Bounty: 1000 $BSQ

**Comment by flix1** [comment]
![image](https://user-images.githubusercontent.com/4233784/71831455-3846ed00-30a9-11ea-8866-a3d2ed62c5ec.png)

There's been some Argentine Peso trades before on Bisq, but not many. This market could definitely use a push. The need is especially great since new restrictions were put on Bitcoin trading in Argentina recently.

**Comment by flix1** [comment]
For examples of similar bounties that were successfully claimed please see #138 and #160

**Comment by flix1** [comment]
Hey @fedegrc I see that you claimed a previous ARS bounty #26. Are you interested in this one?

**Comment by luisantoniocrag** [comment]
Hi 
I am interested in participating, right now I am not in Argentina, in fact I am in Mexico.

But I know many merchants and buyers of BTC in Argentina.

Can I participate for example with western union operations?

And if it is correct, how much is the minimum volume to participate?

**Comment by m52go** [comment]
@luisantoniocrag thanks for your interest, couple of questions.

> right now I am not in Argentina, in fact I am in Mexico.

No problem. Do you have an idea of how you'll be reaching out to your contacts in Argentina? I wonder if/how we could help with your efforts.

Also, an idea: @MwithM mentioned some Latin American countries use USD a lot, and that perhaps folks in those countries could post USD offers instead of bootstrapping their own currencies. Not sure if this makes any sense for Argentina?

> Can I participate for example with western union operations?

It seems all past ARS transactions were settled by national bank transfer, but I realize that's probably difficult to do if you're not in Argentina. Just wanted to point that out.

> how much is the minimum volume to participate?

flix1's original post indicates the goal is to provide 2 buy offers and 2 sell offers. It seems almost all past trades were for 0.01 BTC, so that might might be a good trade size, but you probably know the market better than me.

**Comment by mpolavieja** [comment]
I think Argentina is a must market for Bisq.  I know from facebook bitcoin group in Argentina (more than 40,000 members) there is a lot of activity buying and selling bitcoin, and many people want to do it p2p, and they indeed do it just based on trust and other member´s references.   I am willing to help users and market makers.

Many argentinians have accounts in the US, so they could access the BTC-USD pair, it could be a good way to let them try Bisq in a more liquid market (less frustrating), and once they make a few transactions and get motivated, they might be more willing to participate in BTC-ARS (or even make some basic arbitrage given USD restrictions on the country)

**Comment by MwithM** [comment]
> I think Argentina is a "must have" market for Bisq. I know from facebook bitcoin group in Argentina (more than 40,000 members) that there is a lot of activity buying and selling bitcoin, and many people want to do it p2p, and they indeed do it just based on trust and other member´s references.

Bringing a laptop to a trade meeting is a barrier in secure countries. In Argentina this barrier must be higher. If we want F2F trading to be a real thing at Bisq, laptops should stay home (smartphone app could achieve that) and maybe we would need to have more flexibility with deposits for different kind of transactions (high deposits, over 100% for Mutual Assured Destruction and low deposits (15%) for OpenDime BTC payments).

> Many argentinians have accounts in the US, so they could access the BTC-USD pair, it could be a good way to let them try Bisq in a more liquid market (less frustrating), and once they make a few transactions and get motivated, they might be more willing to participate in BTC-ARS (or even make some basic arbitrage given USD restrictions on the country)

Yes, that's it. Study and explain how to trade USD from Argentina to create happy Bisq users who then will try to trade ARS.

**Comment by mpolavieja** [comment]
> (smartphone app could achieve that)

As a "dirty" alternative while the mobile app is not available, you can use VNC in your laptop and control it from your mobile.  I access Bisq like this very often.  Is not very comfortable, but it works.

**Comment by flix1** [comment]
> And if it is correct, how much is the minimum volume to participate?

There is no minimum volume. As you can see the requirements listed for this bounty are keeping offers online for 1 month and completing 5 trades.

However the point is to kickstart a market by attracting real traders. We want a good faith effort... not somebody just placing 0.0001 BTC offers and then trading with a friend or with himself to complete the 5 trade minimum.

Ideally this activity should be profitable for you even without the bounty. So talk to other traders and if you think that there will be a real market for your chosen payment method... go ahead and give it a try.

Alternatively, since you are in Mexico, you could try to kickstart the **MXN** market which is also important for Bisq. We can create a bounty for MXN/BTC...

**Comment by luisantoniocrag** [comment]
I agree that a local market is better supported, so, I will support the MXN market while in Mexico.

**Comment by flix1** [comment]
@luisantoniocrag see #184

**Comment by mpolavieja** [comment]
Fyi, I have opened an issue regarding Moneygram not working for ARS/BTC. https://github.com/bisq-network/bisq/issues/4000

@m52go and myself are doing some interesting research and real tests with Western Union and other payment methods.   We will report when we finish, but it looks interesting.

**Comment by atom-mota** [comment]
Hi there, I've read the conditions for the bounty, and flix1's comments about "..good faith effort" .  what I wanted to know is if this bounty is accessible to 1 or all that meet the required criteria?
thank you!

**Comment by m52go** [comment]
@atom-mota typically 1 person claims the bounty and owns it. Do you have others you're looking to collaborate with?
